tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -27,37 +28,57 @@
     "sha512 asimdfhm dit uscat ilrcpc flagm ssbs dcpodp i8mm bf16 dgh rng"
 )
 
-DEFAULT_G3_FEATURES_NO_SSBS = (
+DEFAULT_G3_FEATURES_NO_SSBS_4_14 = (
     "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
     "asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 sm3 sm4 asimddp "
     "sha512 asimdfhm dit uscat ilrcpc flagm dcpodp i8mm bf16 dgh rng"
 )
 
+DEFAULT_G3_FEATURES_WITH_SVE_AND_PAC_4_14 = (
+    "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
+    "asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 sm3 sm4 asimddp "
+    "sha512 asimdfhm dit uscat ilrcpc flagm ssbs"
+)
+
+DEFAULT_G3_FEATURES_NO_SSBS_5_10 = (
+    "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
+    "asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 sm3 sm4 asimddp "
+    "sha512 asimdfhm dit uscat ilrcpc flagm dcpodp i8mm bf16 dgh rng"
+)
+
+DEFAULT_G3_FEATURES_WITH_SVE_AND_PAC_5_10 = (
+    "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
+    "asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 sm3 sm4 asimddp "
+    "sha512 sve asimdfhm dit uscat ilrcpc flagm ssbs paca pacg dcpodp svei8mm svebf16 i8mm bf16 dgh rng"
+)
+
 DEFAULT_G3_FEATURES_V1N1 = (
     "fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp "
     "asimdhp cpuid asimdrdm lrcpc dcpop asimddp ssbs"
 )
 
 
-def _check_cpu_features_arm(test_microvm, template_name=None):
+def _check_cpu_features_arm(test_microvm, kernel, template_name=None):
```

Comment:
nit: It would be better to clarify whether the `kernel` means guest or host.

Reply:
changed to `guests_kv`